### ISSUE TITLE
Fix the link to join the leaderboard on the AoC platform

### ISSUE THREAD

**Issue by pil0u** [pil0u/lewagon-aoc#19]
On step 2 of the sync process, the `click here` link is an external form submission that is supposed to make the user join the leaderboard on adventofcode.com.

It was supposed to improve UX, but for some (probably good) reasons, some browsers seem to block that behaviour.

- [ ] Find the root cause
- [ ] Fix

**Comment by pil0u** [comment]
Some users use MacOS + Chrome, but a very few others are from Windows or using another browser.

I still have no clue about why this happens.

**Comment by andreamazzatxt** [comment]
Can be a CORS problem? I'm trying to make the post request from the chrome console and gives me a CORS error
![image](https://user-images.githubusercontent.com/73519803/142680199-b2832070-05af-4a84-90bc-e1c3ed2e0e80.png)

same request from aoc works 
![image](https://user-images.githubusercontent.com/73519803/142680161-84ed6b66-64d4-4b80-9ca9-9375daa86a7c.png)

seems that this aoc endpoint doesn't accept external connections.

**Comment by pil0u** [comment]
Thanks for the investigation, I was not able to reproduce on my side.

I could mess around with the app CORS configuration, I opted for showing the users the code to manually enter if the request was blocked by their browser.

So I `wontfix` the link anytime soon, but I added explanations for step 2 on the "unsynced" Dashboard as a workaround with #21.
